contrib/scale-kind-cluster.sh

```
@@ -0,0 +1,475 @@
+    while [ $COUNT -lt $MAX_WAIT ]; do
+        NODE_STATUS=$(kubectl get node "$NODE_NAME" -o jsonpath='{.status.conditions[?(@.type=="Ready")].status}' 2>/dev/null || echo "")
+        if [ "$NODE_STATUS" = "True" ]; then
+            echo "Node is Ready"
+            break
+        fi
+        sleep 3
+        COUNT=$((COUNT + 3))
+    done
+
+    if [ "$NODE_STATUS" != "True" ]; then
+        echo "Node not Ready after ${MAX_WAIT}s (current status: $(kubectl get node $NODE_NAME --no-headers 2>/dev/null | awk '{print $2}' || echo 'Unknown'))"
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)